lldb/source/Plugins/Language/ObjC/NSSet.cpp

```diff
@@ -227,22 +221,6 @@ namespace Foundation1437 {
   }
 }
   
-class NSSetCodeRunningSyntheticFrontEnd : public SyntheticChildrenFrontEnd {
-public:
-  NSSetCodeRunningSyntheticFrontEnd(lldb::ValueObjectSP valobj_sp);
-
-  ~NSSetCodeRunningSyntheticFrontEnd() override;
-
-  llvm::Expected<uint32_t> CalculateNumChildren() override;
-
-  lldb::ValueObjectSP GetChildAtIndex(uint32_t idx) override;
-
-  lldb::ChildCacheState Update() override;
-
-  bool MightHaveChildren() override;
```

Comment:
while auditing `MightHaveChildren` declarations, the name of this class caught my attention. I looked for uses and found none, so I've removed it.